styles.css

```diff
@@ -728,3 +754,666 @@ a {
+    display: flex;
+    align-items: center;
+    color: #1c1a19;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .recent_work_content {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 0;
+    gap: 2px;
+    width: 1156px;
+    height: 448px;
+    left: -1px;
+    top: 0;
+  }
+
+  .recent_work_language {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: 314px;
+    height: 40px;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .img_placeholder {
+    width: 684px;
+    height: 448px;
+    margin: 0 40px 0 0;
+  }
+
+  .placeholder_content {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    width: 448px;
+    height: 348px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .multi_story {
+    width: 448px;
+    height: 48px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    margin-top: -10px;
+  }
+
+  .see_prjt_buttn {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+    background: #f55800;
+    flex: none;
+    order: 3;
+    flex-grow: 0;
+    margin-top: 5px;
+  }
+
+  .recent_work_text {
+    width: 448px;
+    height: 140px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+    margin-top: -5px;
+  }
+
+  .recent_project {
+    display: grid;
+    gap: 24px;
+    margin-top: 10rem;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .card_bttn {
+    background: #ff6b00;
+    padding: 12px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    border: none;
+    color: white;
+    cursor: pointer;
+    width: 100%;
+    z-index: 10;
+    transition: all 0.5s ease-in-out;
+    opacity: 0;
+  }
+
+  .project_background {
+    position: relative;
+    background: url('image folder/projects-background.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 95%;
+    height: 395px;
+  }
+
+  .project_foreground {
+    position: relative;
+    background:
+      linear-gradient(
+        179.35deg,
+        rgba(38, 38, 38, 0) 0.85%,
+        rgba(38, 38, 38, 0.9) 84%
+      );
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    transition: transform 600ms cubic-bezier(0.73, 0.59, 0, 0.55);
+  }
+
+  .project_foreground:hover {
+    transform: scale(1.01);
+    transform-origin: 0 0;
+    z-index: 10000;
+    opacity: 0;
+  }
```

Comment:
Good job on making your cards interact with hover effects 💪, but one important detail is missing the see project button should appear when hovering 

**figma design**
![see project button on hover](https://user-images.githubusercontent.com/59707859/229291407-e43bc1d5-35aa-4b8f-b994-eaece81d25dd.PNG)

Reply:
Thank you @souad988 for your review, concerning your last comment on adding hover effect to the see project button, I'm not sure what I am not doing correctly, i am unable to make the button to show when you hover, can you assist? I will like a suggestion if any on how to resolve it. Thanks